.github/workflows/pull_request_secure.yml

```diff
@@ -287,13 +281,60 @@ jobs:
           node-version-file: .nvmrc
           cache: npm
 
+      - name: Checkout timefold-solver
+        uses: actions/checkout@v4
+        with:
+          repository: "${{ github.event.pull_request.head.repo.owner.login || 'TimefoldAI' }}/timefold-solver"
+          ref: ${{ github.event.pull_request.head.sha || 'main' }} # The GHA event will pull the main branch by default, and we must specify the PR reference version
+          path: "./timefold-solver"
+          fetch-depth: 0
+
+      - name: Install yq
+        run: |
+          sudo wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq 
+          sudo chmod +x /usr/bin/yq          
+
+      - name: Configuring Antora version
+        working-directory: "./timefold-solver"
+        run: |
+          LATEST_VERSION="SNAPSHOT"
+          echo "LATEST_VERSION=$LATEST_VERSION" >> "$GITHUB_ENV"
```

Comment:
I'm thinking maybe we do the `sed` right here, instead of first sending the information to env and then reading it from there?

It's a lot of boilerplate already, and I don't think the extra script makes it easier to read or maintain.